src/components/Home/Home.js

```diff
@@ -63,12 +69,13 @@ export default class Home extends React.Component {
           {(this.state.filteredMovies.length || this.state.error === 'No movies found.') 
             ? <Preview className='closed'/> : <Preview className='opened' />
           }
-          {!this.state.error && !this.state.movies.length && 
+          {this.state.fetchedError && apiCalls.checkForError(this.state.statusCode)}
```

Comment:
Glad that we are now using this method to render our error messages to the user!